### PR TITLE
ci: retry unstable steps.

### LINKED ISSUE #13: Retry the commands that are most likely to fail on travis
Check which commands are more likely to fail (usually the ones that depend on external services, specially building the images as it pulls stuff from a bunch of different repositories) and retry them on failure so it's less like to break the build

### PR DESCRIPTION
**Pull request by david-caro**
* Retrying some of the steps that have failed by 'temporary' external
  issues (like net/timeouts) (fixes #13)

Signed-off-by: David Caro <dcaroest@redhat.com>

### CODE REVIEW

**Comment by david-caro** [comment]
What do you mean by harsh?

**Comment by mihaibivol** [comment]
@david-caro should we merge this before #15 ?

File: README.md [hunk 1]
```diff
@@ -25,3 +25,56 @@ the Overlay do:
 ```shell
 docker pull inspirehep/elasticsearch:latest
 ```
+
+
+## Testing changes to this repo
+
+Inorder to test any changes you do to the Dockerfiles in this repo, you can
```
Comment (mihaibivol) [inline README.md]: :nit: In order

**Comment by david-caro** [comment]
Should be independent yes, though it might need rebasing #15 after. Let me fix the typo and we can merge

**Pull request merged**